Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -242,13 +274,22 @@ extension ExperimentalAnimationLayer: RootAnimationLayer {
           closure: nil),
         timingConfiguration: CAMediaTimingConfiguration(speed: 0))
 
-      if currentAnimationConfiguration != requiredAnimationConfiguration {
-        setupAnimation(
+      if pendingAnimationConfiguration != nil {
+        pendingAnimationConfiguration = (
+          animationConfiguration: requiredAnimationConfiguration,
+          playbackState: .paused(frame: newValue))
```

Comment:
Do we not need a `setNeedsDisplay` here to flush this `pendingAnimationConfiguration`? Might be helpful to have an explanatory comment